Makefile

```diff
@@ -493,7 +493,7 @@ go-clean: ## go clean build, test and modules caches
 
 .PHONY: help
 help:
-	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[.0-9A-Za-z_-]+:.*?##/ { printf "  \033[36m%-45s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
```

Comment:
when running `make` on `main` locally i was getting : 
```
make             
Makefile                       go build
Makefile                       CI build
Makefile                       remove files created during build
Makefile                       dev build
Makefile                       pushes the devkit-image
Makefile                       git diff
Makefile                       go generate
Makefile                       go clean build, test and modules caches
Makefile                       install envsubst binary
Makefile                       go test with race detector for integration tests
Makefile                       golangci-lint
Makefile                       go mod tidy
Makefile                       run all linting with super-linter
Makefile                       open a shell in the super-linter container
Makefile                       go test with race detector and code coverage
test/infra/aws/Makefile        Download and unpack terraform binary
```
so i copied this recipe from konvoy repo which fixed the output for me.